spec/wrappers/prospect_api_wrapper_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+RSpec.describe ProspectApiWrapper do
+  describe '.get_prospects' do
+    it 'returns prospect attributes from API' do
+      stub_const 'ProspectApiWrapper::API_KEY', 'abc'
+      allow(RestClient).to receive(:get)
```

Comment:
* for starters, prefer keeping `.` in trailing line (to indicate "hey, there will be more")
* indent following line 1 tab 

```
expect(UpsertBookingFromInquiry).to have_received(:call).with(
  inquiry, reservation_details
)
```

```
expect(Streamline::SyncInquiry).to have_received(:call).
  with(inquiry_1, details_1)
```

* break lines on first parameter

```
.with('http://prospects-api.herokuapp.com/prospects',
                                    { params: { api_key: 'abc' } })
```
->
```
.with(
  'http://prospects-api.herokuapp.com/prospects',
  { params: { api_key: 'abc' } }
)
```